<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>935</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go Fish</string>
  </property>
  <property name="locale">
   <locale language="English" country="Canada"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="lyout_bot_output_out">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_bot_output">
        <layout class="QHBoxLayout" name="lyout_bot_output_in">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Bot 1 Output</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPlainTextEdit" name="te_output_bot1">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Bot 2 Output</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QPlainTextEdit" name="te_output_bot2">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Bot 3 Output</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPlainTextEdit" name="te_output_bot3">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="lyout_bottom">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="hand_cards_layout">
        <item>
         <widget class="QGroupBox" name="cards_hand_grp">
          <property name="title">
           <string>Cards in Hand</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="card_hands_layout">
           <item>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>&amp;Card 1</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="card_1">
             <property name="text">
              <string>Card &amp;2</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Card &amp;3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_4">
             <property name="text">
              <string>Card &amp;4</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>Remaining Fish</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QLCDNumber" name="lcd_cards_remaining">
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="bot_btn_layout">
        <item>
         <widget class="QGroupBox" name="grp_ask">
          <property name="title">
           <string>Bot to Ask</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="lyout_ask">
           <item>
            <widget class="QPushButton" name="btn_ask_bot1">
             <property name="text">
              <string>Bot 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_ask_bot2">
             <property name="text">
              <string>Bot 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_ask_bot3">
             <property name="text">
              <string>Bot 3</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grp_hands">
          <property name="title">
           <string>Cards in Bot hands</string>
          </property>
          <layout class="QVBoxLayout" name="lyout_hands">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Bot 1</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLCDNumber" name="lcd_cards_bot1">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>Bot 2</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLCDNumber" name="lcd_cards_bot2">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string>Bot 3</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLCDNumber" name="lcd_cards_bot3">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_pairs">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="lyout_pairs">
         <item row="0" column="0">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Your Pairs</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Score:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="score_player">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="te_pairs_player">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Bot 1 Pairs</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Score:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="score_bot1">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="te_pairs_bot1">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Bot 3 Pairs</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Score:</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLCDNumber" name="score_bot3">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="te_pairs_bot3">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Bot 2 Pairs</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Score:</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLCDNumber" name="score_bot2">
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="te_pairs_bot2">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grp_player_output">
        <property name="title">
         <string>Player Output</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0">
         <item>
          <widget class="QPlainTextEdit" name="te_player_output">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionSave_Game"/>
    <addaction name="actionOpen_Game"/>
    <addaction name="actionPrint_hand_to_JSON"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave_Game">
   <property name="text">
    <string>&amp;Save Game</string>
   </property>
  </action>
  <action name="actionOpen_Game">
   <property name="text">
    <string>&amp;Open Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
  </action>
  <action name="actionPrint_hand_to_JSON">
   <property name="text">
    <string>Print hand to JSON</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
